test/e2e/reboots/kernel_parameter_add_rm.go

```diff
@@ -83,9 +83,17 @@ var _ = ginkgo.Describe("[reboots][kernel_parameter_add_rm] Node Tuning Operator
 			_, _, err = util.ExecAndLogCommand("oc", "create", "-f", mcpRealtime)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-			ginkgo.By("waiting for worker-rt MachineConfigPool UpdatedMachineCount == 1")
-			err = util.WaitForPoolUpdatedMachineCount(cs, "worker-rt", 1)
-			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			waitForMCPFlip := func() {
```

Comment:
Can we define this function somewhere else (e.g. test/e2e/util/util.go) and call it in all the spots this is being done in this PR?

Reply:
Not sure its usefulness warrants placement in `util.go`.  Perhaps once we want to use it in another test.

Reply:
Thank you for looking, David.  I don't think this function has a place in `test/e2e/util/util.go`.  I don't think I want to see any `ginkgo.By()` and `gomega.Expect()` there.   Moving to `operator_test.go`, hopefully you're fine with that.

Reply:
That makes sense, thanks. 

I wanted to avoid duplicating these ~6 lines more than a couple times, and this accomplishes that.